content/shmem_sync_all.tex

```diff
@@ -0,0 +1,44 @@
+\color{ForestGreen}
+\apisummary{
+    Registers the arrival of a \ac{PE} at a barrier and suspends \ac{PE}
```

Comment:
Without considering threads, what should be behavior of non-blocking operations during "suspend" ?